core/src/main/scala/org/apache/spark/broadcast/TorrentBroadcast.scala

```diff
@@ -78,6 +80,7 @@ private[spark] class TorrentBroadcast[T: ClassTag](obj: T, id: Long)
     }
     // Note: use getSizeAsKb (not bytes) to maintain compatibility if no units are provided
     blockSize = conf.getSizeAsKb("spark.broadcast.blockSize", "4m").toInt * 1024
```

Comment:
`spark.broadcast.blockSize` has special meaning. I don't think we should replace it with `pageSizeBytes`.